package-lock.json

```diff
@@ -1,19868 +1,6 @@
 {
```

Comment:
Can you revert changes to this file? It should already be setup right. You shouldn't need to override it like this.

Reply:
I can not find this file for my BorrowItem

Reply:
Okay i reverted it to pre commit